internal/locale/locales/en-us.yaml

```diff
@@ -1718,6 +1718,8 @@ cve_needs_authentication:
   other: You need to be authenticated in order to access vulnerability information about your project.
 auth_tip:
   other: Run `state auth` to authenticate.
+err_non_interactive_terminal:
+  other: You are running the State Tool from a non-interactive terminal, but interactive input was needed. Please re-run the State Tool from an interactive terminal. If you are using Git Bash on Windows for example, prefix your command with [ACTIONABLE]`winpty`[/RESET].
```

Comment:
Can we link to some online article explaining the issue being worked around here?